generator/postgres/query_set.go

```diff
@@ -89,3 +92,69 @@ ORDER BY n.nspname, t.typname, e.enumsortorder;`
 
 	return result
 }
+
```

Comment:
This function is the same as for postgres, I think we can just `return postgresQuerySet.GetTablesMetaData(db, schemaName, tableType)`

Reply:
Yes but it might be a good idea to keep it independent in case future changes to the postgres query breaks it.

Reply:
In case of future breaking changes there will be tests that will catch if something breaks.

Reply:
Updated.